packages/frontend/src/state/reducers/modals/actions/index.ts

```diff
@@ -0,0 +1,5 @@
+import { createAction } from "@reduxjs/toolkit";
+
+export const setModalIsOpen = createAction<{
```

Comment:
I'd rename this `setModalOpen`.

Reply:
fixed: 133bb15676195f8a8a2cdb417ac0dc5fc4172360